django/src/rdwatch_scoring/views.py

```diff
@@ -0,0 +1,216 @@
+    site_preparation: str = None
+    active_construction: str = None
+    post_construction: str = None
+
+
+class EvaluationResponseSchema(Schema):
+    region_name: str = None
+    evaluationId: int = None
+    evaluation_run: int = None
+    performer: str = None
+    evaluation_uuid: str = None
+    unionArea: float = None
+    statusAnnotated: str = None
+    temporalIOU: TemporalSchema = None
+    color: str = None
+
+
+# If the combination of configurationId and regionId has scoring
+@router.get('/has_scores')
+def has_scores(request: HttpRequest, configurationId: int, regionId: int):
+    # from the hyper parameters we need the evaluation and evaluation_run Ids
+    configuration = get_object_or_404(HyperParameters, pk=configurationId)
+    evaluationId = configuration.evaluation
+    performer_name = configuration.performer.slug.lower()
+    evaluation_run = configuration.evaluation_run
+    region_name = RegionSchema.resolve_name(Region.objects.filter(pk=regionId).first())
+
+    return EvaluationRun.objects.filter(
+        evaluation_run_number=evaluation_run,
+        evaluation_number=evaluationId,
+        region=region_name,
+        performer=performer_name,
+    ).exists()
+
+
+# Region scoring Map
+@router.get('/region_scoring_colors')
+def region_color_map(request: HttpRequest, configurationId: int, regionId: int):
+    # from the hyper parameters we need the evaluation and evaluation_run Ids
+    configuration = HyperParameters.objects.filter(pk=configurationId).first()
+    evaluationId = configuration.evaluation
+    performer_name = configuration.performer.slug.lower()
+    evaluation_run = configuration.evaluation_run
+    region_name = RegionSchema.resolve_name(Region.objects.filter(pk=regionId).first())
+
+    evaluation = EvaluationRun.objects.filter(
+        evaluation_run_number=evaluation_run,
+        evaluation_number=evaluationId,
+        region=region_name,
+        performer=performer_name,
+    ).first()
+    if evaluation:
+        evaluationUUID = evaluation.uuid
+        # Now we can get a list of the  SiteIds information for this evaluationId
+        site_list = Site.objects.filter(
+            region_id=region_name,
+            evaluation_run_uuid=evaluationUUID,
+            originator=performer_name,
+        )
+        colorMappers = {}
+        for _i, site in enumerate(site_list):
+            site_number = int(
+                site.site_id.replace(f'{region_name}_', '').replace(
+                    f'_{performer_name}_{site.version}', ''
+                )
+            )
+            colorMappers[
+                f'{configurationId}_{regionId}_{configuration.performer.id}_{site_number}'  # noqa: E501
+            ] = get_site_scoring_color(evaluationUUID, 'overall', site.site_id)
+
+    return colorMappers
+
+
+# Gets the basic scoring information based on the
+# configurationId, regionId SiteNumber and Version
+@router.get('/scoring_details', response=EvaluationResponseSchema)
```

Comment:
Considering the full endpoint URL will be `/api/scores/scoring_details`, what do you think about removing the reference to scoring here?
```suggestion
@router.get('/details')
```
Or perhaps even
```suggestion
@router.get('/')
```